### PR TITLE
Fix autoscaler process width calculation

### PR DESCRIPTION
**Pull request by ddollar**

### CODE REVIEW

**Comment by nzoschke** [comment]
The 5s tick concerns me about rate limiting and throttling.

**Comment by ddollar** [comment]
Oh my bad that was just for debugging, will fix

File: api/workers/autoscale.go [hunk 1]
```diff
@@ -72,10 +75,7 @@ func autoscaleRack() {
 
 	err = models.Provider().SystemSave(*system)
 	if err != nil {
-		log.Log("fn=system.Save err=%q", err)
+		log.Error(err)
 		return
 	}
-
-	// log for humans
-	fmt.Printf("who=\"convox/monitor\" what=\"autoscaled instance count to %d\" why=\"a service wants %s processes behind a load balancer\"\n", system.Count)
```
Comment (nzoschke) [inline api/workers/autoscale.go]: These have been helpful for debugging... The goal is to have a simple rack logs filter we can document so users can understand some internal changes that are executed and why

**Comment by nzoschke** [comment]
There is an issue around this. Can you also review the memory calculation while you are in here? I think we want to add one extra instance because we can't always perfectly pack for memory

**Comment by ddollar** [comment]
The +1 for extra room has already been in for quite some time.

**Comment by ddollar** [comment]
This PR also now takes CPU into account for scaling and displays the autoscaling reason.

To search for autoscale events you can `--filter="autoscaleRack change="`

File: api/workers/autoscale.go (hunk 1 above)
Comment (ddollar) [inline api/workers/autoscale.go]: This message was misleading. Autoscale can happen for more reasons than just process count. It has been replaced by better logs.

**Comment by ddollar** [comment]
Just a note of order. Line-based comments are better if possible because you can "clear" them when the code changes.

**Comment by mwarkentin** [comment]
Might be nice to have a note in the [AutoScale docs](https://convox.com/docs/scaling/) about how to search for autoscale events?

**Comment by ianmalott** [comment]
@mwarkentin 👍 adding a note now

**Comment by mwarkentin** [comment]
Thanks!

**Pull request closed without merging**